docker/.env.example

```diff
@@ -30,6 +30,8 @@ WREN_BOOTSTRAP_VERSION=0.1.5
 # AI service related env variables
 AI_SERVICE_ENABLE_TIMER=
 AI_SERVICE_LOGGING_LEVEL=INFO
+SHOULD_FORCE_DEPLOY=true
```

Comment:
I think it is better to enable it manually. what do you think about setting it false as default?